onbuild/Dockerfile

```diff
@@ -0,0 +1,29 @@
+#
+# Dockerfile for Storm Mesos framework
+#
+FROM mesosphere/mesos:0.27.0-0.2.190.ubuntu1404
```

Comment:
Ah, @lfundaro, I lazily responded before reading your [earlier message of the day](https://github.com/mesos/storm/pull/109#issuecomment-200314973").  So, having varying dirs for Dockerfiles for each mesos version is another potential solution/workaround for handling mesos version differences, and probably more "canonical" Docker-workflow-wise.  What say you @DarinJ & @dsKarthick & @brndnmtthws?